html/comparison.html.haml

```diff
@@ -428,7 +428,7 @@
                 %span.summary-part{"ng-show" => "diff.repositories.only_in2"}
                   {{diff.meta.description_b}}: {{diff.repositories.only_in2.length}} repos
                 %span.summary-part{"ng-show" => "diff.repositories.common"}
-                  %a{:class=>"show-common-elements", href: "#repositories_both"}
+                  %a{:class=>"show-common-elements", href: "#repositories_both"}<
```

Comment:
Line is too long. [81/80]<br>Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace